### PR TITLE
p2p: possibly fix crash in relay_blocks

### PR DESCRIPTION
**Pull request by moneromooo-monero**

### CODE REVIEW

**Comment by NanoAkron** [comment]
Do you think this is what's been causing us to crash recently?

**Comment by moneromooo-monero** [comment]
As the message says... "possibly".

**Comment by NanoAkron** [comment]
;)

File: src/p2p/net_node.h [hunk 1]
```diff
@@ -172,6 +172,7 @@ namespace nodetool
     virtual void on_connection_close(p2p_connection_context& context);
     virtual void callback(p2p_connection_context& context);
     //----------------- i_p2p_endpoint -------------------------------------------------------------
+    virtual bool relay_notify_to(int command, const std::string& data_buff, const std::list<boost::uuids::uuid> &connections);
```
Comment (tewinget) [inline src/p2p/net_node.h]: could probably call this "relay_notify_to_multiple" or similar, just to make for easier understanding on a cursory glance.

Comment (moneromooo-monero) [inline src/p2p/net_node.h]: There's a list of peers as last parameter, so it is intended to read "relay to ... this list".

Comment (moneromooo-monero) [inline src/p2p/net_node.h]: Why would it not make sense in 5 years ?

Comment (NanoAkron) [inline src/p2p/net_node.h]: The word 'multiple' has an explicit meaning rather than expecting future devs to derive it implicitly.

Comment (moneromooo-monero) [inline src/p2p/net_node.h]: I guess. Like memcpy_ptr_to_ptr. Nothing to do with time, then. Still, the list might be 0 or 1 long. So multiple actually makes things less clear. Maybe _to_list ?

**Comment by tewinget** [comment]
List seems reasonable.

File: src/p2p/net_node.h (hunk 1 above)
Comment (NanoAkron) [inline src/p2p/net_node.h]: Nice ;) Apologies for the bike-shedding...

**Comment by NanoAkron** [comment]
Currently testing

**Pull request merged**